src/web/client/extension.ts

```diff
@@ -71,7 +69,10 @@ export function activate(context: vscode.ExtensionContext): void {
                                 context.globalState.update(IS_FIRST_RUN_EXPERIENCE, false);
                                 WebExtensionContext.telemetry.sendInfoTelemetry("StartCommand", { 'commandId': 'workbench.action.openWalkthrough', 'walkthroughId': 'microsoft-IsvExpTools.powerplatform-vscode#PowerPage-gettingStarted' });
                             }
-
+                            await NPSService.setEligibility();
```

Comment:
We should do this after data initialization  that happens by the readDirectory call below.